internal/pkg/dependencies/project.go

```diff
@@ -42,6 +42,11 @@ func NewProjectDeps(ctx context.Context, base Base, public Public, tokenStr stri
 }
 
 func newProjectDeps(base Base, public Public, token storageapi.Token) (*project, error) {
+	// Require master token
+	if !token.IsMaster {
+		return nil, errors.New("a master token of a project administrator is required")
+	}
+
```

Comment:
E2E test na to schvalne nie je, neoplati sa nam v kazdom testovacom projekte vytvarat non-master token, pre jeden test/jednu hlasku.